From: Jory A. Pratt <anarchy@gentoo.org>
Modified: Luca Giambonini <almack@chakralinux.org>

Enable to build with LTO support

diff --git a/security/sandbox/linux/moz.build b/security/sandbox/linux/moz.build
--- a/security/sandbox/linux/moz.build
+++ b/security/sandbox/linux/moz.build
@@ -100,7 +100,7 @@
 # from the function using it which breaks the build.  Work around that by
 # forcing there to be only one partition.
 if '-flto' in CONFIG['OS_CXXFLAGS'] and CONFIG['CC_TYPE'] != 'clang':
-    LDFLAGS += ['--param lto-partitions=1']
+    LDFLAGS += ['--lto-partition=one']
 
 DEFINES['NS_NO_XPCOM'] = True
 DisableStlWrapping()
